src/TwigComponent/src/DependencyInjection/TwigComponentExtension.php

```diff
@@ -111,22 +109,13 @@ class_exists(AbstractArgument::class) ? new AbstractArgument(sprintf('Added in %
             ->setDecoratedService(new Reference('twig.configurator.environment'))
             ->setArguments([new Reference('ux.twig_component.twig.environment_configurator.inner')]);
 
-        $container->register('ux.twig_component.command.debug', TwigComponentDebugCommand::class)
-            ->setArguments([
-                new Parameter('twig.default_path'),
-                new Reference('ux.twig_component.component_factory'),
-                new Reference('twig'),
-                class_exists(AbstractArgument::class) ? new AbstractArgument(sprintf('Added in %s.', TwigComponentPass::class)) : [],
-                $config['anonymous_template_directory'],
-            ])
-            ->addTag('console.command')
-        ;
-
         $container->setAlias('console.command.stimulus_component_debug', 'ux.twig_component.command.debug')
             ->setDeprecated('symfony/ux-twig-component', '2.13', '%alias_id%');
 
         if ($container->getParameter('kernel.debug')) {
             $loader->load('debug.php');
+
+            $container->setParameter('ux.twig_component.anonymous_template_directory', $config['anonymous_template_directory']);
```

Comment:
Not sure to understand why we need this ? 

i'd rather not set container parameters (even in debug only) if we can avoid

Reply:
> Not sure to understand why we need this ?

This was a relic from the previous PR. I think I can remove.